cmd/format-xl.go

```diff
@@ -330,6 +330,7 @@ func loadFormatXLAll(storageDisks []StorageAPI) ([]*formatXLV3, []error) {
 			if err != nil {
 				return err
 			}
+			storageDisks[index].SetDiskID(format.XL.This)
```

Comment:
I think it is better if we move this next to loadFromXLAll() instead of inside this latter

Reply:
With lots of disks, large loops are costly, we cannot do that @vadmeste